### ISSUE TITLE
FSEQ will not load to ESPstick from the file manager and files are not seen that are on the disk

### ISSUE THREAD

**Issue by rbearwy** [forkineye/ESPixelStick#368]
--------- Instructions --------
Please provide answers directly below each section.
--------- Instructions ---------

**ESPixelStick Firmware Version**
Version 4.02Beta

**Hardware Version**

**Binary release or compiled yourself?**

**Operating System (and version)**

Windows 10
**Web Browser (and version)**
Chrome

**Access Point**
wifi

**Describe the bug**
I tried loading FSEQ files after flashing the update that was just released.  It says it loaded, yet nothing comes up in the file management in the control setup.  I had it set to FPP remote when doing this.  I powered down the stick and removed the SD card and checked.  The FSEQ files are all there they are just not recognized by the device.  The original two that I downloaded back in June are there and show fine.  I tried to run the stick using the fppmaster and it would not play.  I then ran the fppmaster with one of the two FSEQ files that I could see and they ran fine.

**Comment by MartinMueller2003** [comment]
Please attach one or two of the files you tried to download. I just downloaded a set of files and did not have an issue.

**Comment by rbearwy** [comment]
The window below told me that it does not support that file type when I tried to drop my fseq file below

**Comment by rbearwy** [comment]
[bobby_pickett_monster_mash.zip](https://github.com/forkineye/ESPixelStick/files/7261049/bobby_pickett_monster_mash.zip)

**Comment by forkineye** [comment]
> [bobby_pickett_monster_mash.zip](https://github.com/forkineye/ESPixelStick/files/7261049/bobby_pickett_monster_mash.zip)

Those are compressed files, only sparse V2 uncompressed files are supported.

```
C:\Users\sporadic\Desktop\test>python fseqinfo.py *.fseq
Beetlejuice - Day-o.fseq:
  V2.0 compressed (zstd) 136780 channels with 2353 frames @ 50ms from xLights Windows 2021.29 64bit

bobby_pickett_monster_mash.fseq:
  V2.0 compressed (zstd) 136780 channels with 3522 frames @ 50ms from xLights Windows 2021.29 64bit
```

**Comment by rbearwy** [comment]
On Thursday, September 30, 2021, 09:40:51 AM MST, RonB ***@***.***> wrote:  
 
  

    On Thursday, September 30, 2021, 09:33:28 AM MST, RonB ***@***.***> wrote:  
 
  

    On Thursday, September 30, 2021, 09:29:41 AM MST, Shelby Merrick ***@***.***> wrote:  
 
 



bobby_pickett_monster_mash.zip


Those are compressed files, only sparse V2 uncompressed files are supported.
C:\Users\sporadic\Desktop\test>python fseqinfo.py *.fseq
Beetlejuice - Day-o.fseq:
  V2.0 compressed (zstd) 136780 channels with 2353 frames @ 50ms from xLights Windows 2021.29 64bit

bobby_pickett_monster_mash.fseq:
  V2.0 compressed (zstd) 136780 channels with 3522 frames @ 50ms from xLights Windows 2021.29 64bit

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by rbearwy** [comment]
I sent back a file through my email

**Comment by rbearwy** [comment]
Github is rejecting the others as two big one is 6.5mb

**Comment by forkineye** [comment]
> Github is rejecting the others as two big one is 6.5mb

You can send them to shelby@forkineye.com. I'll get them to Martin as well if needed.

**Comment by rbearwy** [comment]
Ok sent them to shelby

**Comment by forkineye** [comment]
> Ok sent them to shelby

Got them. Those are compressed as well and look like they're for your entire show. Those files must be sparse uncompressed v2 files which get created and uploaded by FPP connect, which we just found an issue with in beta2.  That is being tracked in issue https://github.com/forkineye/ESPixelStick/issues/367.  

```
C:\Users\sporadic\Desktop\test>python fseqinfo.py -v *.fseq

Beetlejuice - Day-o.fseq
------------------------
File ID:        PSEQ
Data Offset:    2088
Minor Version:  0
Major Version:  2
Header Length:  1936
Channel Count:  136780
Frame Count:    2353
Step Time(ms):  50
Flags:          0
Compression:    1 (zstd)
Comp Blocks:    238
Sparse Ranges:  0
FLags 2:        0
Unique ID:      1632694326446000
Media File:     G:\xlights\Downloads\Extreme Lights\xS_Beetlejuice - Day-O\xS_Beetlejuice - Day-O\_lost\Beetlejuice - Day-o.mp3
Created By:     xLights Windows 2021.29 64bit

john_carpenter_halloween_theme_-4329603140284364213.fseq
--------------------------------------------------------
File ID:        PSEQ
Data Offset:    2096
Minor Version:  0
Major Version:  2
Header Length:  1984
Channel Count:  143492
Frame Count:    2901
Step Time(ms):  50
Flags:          0
Compression:    1 (zstd)
Comp Blocks:    244
Sparse Ranges:  0
FLags 2:        0
Unique ID:      1632614243411000
Media File:     G:\xlights\Audio\john_carpenter_halloween_theme_-4329603140284364213.mp3
Created By:     xLights Windows 2021.29 64bit
```

**Comment by rbearwy** [comment]
So how do I get those files.  Mine are all fseq, xlights program or xbck files.  Is there a way to convert the fseq to v2 files?

**Comment by forkineye** [comment]
> So how do I get those files. Mine are all fseq, xlights program or xbck files. Is there a way to convert the fseq to v2 files?

xLights generates them for you.   They need to be uploaded via FPP Connect once you have your ESPixelStick configured as FPP Remote:
![image](https://user-images.githubusercontent.com/4047923/135504791-9e067d74-a75e-4db6-98d9-cb112a78080f.png)

For standalone props that aren't controlled by your show, or off season effects, etc... You can setup a configuration for just that one prop / controller and change xLights via the preferences and use the fseq that xLights normally saves.  Note though, you don't want to do this for your current setup / show as it will have all the channel data for your show and the uncompressed file will be huge:
![image](https://user-images.githubusercontent.com/4047923/135504558-a4986c82-154e-4a58-a8db-1250af264660.png)

**Comment by rbearwy** [comment]
That was what I was doing and did not see the files in the file manager. I reflashed with the tool and tried it again and no change.

**Comment by forkineye** [comment]
> That was what I was doing and did not see the files in the file manager. I reflashed with the tool and tried it again and no change.

There is a bug I found today causes the files not to show if you refresh your browser while on the device setup tab. You have to navigate to another tab and back.  Would you be available on zoom later?

**Comment by rbearwy** [comment]
yes I can zoom whenever you guys want

**Comment by forkineye** [comment]
I'm on the xLights zoom in breakout room 1 right now.

**Comment by rbearwy** [comment]
going there right now

**Comment by MartinMueller2003** [comment]
FYI: Even if the files inside the zip were uncompressed sparse files, the zip file would still have been rejected. A zip file is NOT an fseq file. You can only upload a file with an FSEQ extension. Monster.fseq, foo.fseq, bar.fseq are all valid. MyFiles.zip will be rejected since the ESP cannot play a zip file.

**Comment by MartinMueller2003** [comment]
Anyone have a link to the room? I get someone giving a tutorial on general usage.
    On Thursday, September 30, 2021, 02:37:32 PM EDT, rbearwy ***@***.***> wrote:  
 
 


going there right now

—
You are receiving this because you were assigned.

Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by patdelaney** [comment]
go to xlights.org  the link is on the right

**Comment by forkineye** [comment]
Issue ended up being the SD card and was very odd. Card originally worked 3 months ago and he was able to upload files to it then. Those files still showed up. He was able to upload files via FPP Connect and via the web interface, but the ESP would not recognize them once written. He pulled the card and was able to see the files he had uploaded in Windows.  Re-inserted card into ESPixelStickV3 and they wouldn't show up.  Windows didn't automatically detect any errors with the card. Ran a force scan and Windows ejected the card saying it couldn't repair it.  Tried a new SD card and everything worked as expected.  Also flashed https://github.com/forkineye/ESPixelStick/actions/runs/1291838875.

**Comment by rbearwy** [comment]
Thank you guys so much for the help.  Your company is truly customer service orientated. Something that is often hard to find in the Brick and mortar businesses
    On Thursday, September 30, 2021, 01:31:54 PM MST, Shelby Merrick ***@***.***> wrote:  
 
 


Closed #368.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by MartinMueller2003** [comment]
Company? I am a Hobiest working on a fun project.

Sent from Yahoo Mail on Android 
 
  On Thu, Sep 30, 2021 at 18:41, ***@***.***> wrote:   

 Thank you guys so much for the help.  Your company is truly customer service orientated. Something that is often hard to find in the Brick and mortar businesses
 On Thursday, September 30, 2021, 01:31:54 PM MST, Shelby Merrick ***@***.***> wrote: 
 
 


Closed #368.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
 
—
You are receiving this because you were assigned.

Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
